client/verta/verta/_internal_utils/_utils.py

```diff
@@ -528,7 +528,12 @@ def raise_for_http_error(response):
         curr_time = timestamp_to_str(now(), utc=True)
         time_str = " at {} UTC".format(curr_time)
 
-        reason = six.ensure_str(response.text.strip())
+        try:
+            reason = six.ensure_str(body_to_json(response)['message'])
+        except (ValueError,  # not JSON response
+                 KeyError):  # no 'message' from back end
+            reason = six.ensure_str(response.text.strip())
```

Comment:
I think the call to `ensure_str()` should be pulled out of the `try-except`

1. both branches use it, so we might as well just call it once, after the fact
2. the `try` block is only testing for a JSON error and a key error—the call to `ensure_str()` is unrelated and so should be pulled out if possible

```suggestion
        try:
            reason = body_to_json(response)['message']
        except (ValueError,  # not JSON response
                 KeyError):  # no 'message' from back end
            reason = response.text.strip()
            
        reason = six.ensure_str(reason)
```

Reply:
Fixed